fhservices.cfg.j2

```diff
@@ -79,26 +79,40 @@ define host {
        hostgroups {{ rhmap_hostgroups }}
 }
 
-{% for fhservice in fh_services if 'ping' in fhservice['checks'] %}
-define service {
-       service_description {{ fhservice['name'] }}::Ping
-       check_command check_fh_component_http_ping!{{ fhservice['name'] }}!{{ fhservice.get("port", "8080") }}!{{ fhservice.get("ping_endpoint", "/sys/info/ping") }}
-       use generic-service
-       hostgroup_name {{ ",".join(fhservice['hostgroups']) }}
-       notes This server cannot access {{ fhservice['name'] }}, check it is running and that this server is able to talk to it on port 8080
-       contact_groups rhmapadmins
-}
-{% endfor %}
-
-{% for fhservice in fh_services if 'health' in fhservice['checks'] %}
-define service {
-       service_description {{ fhservice['name'] }}::Health
-       check_command check_fh_component_health!{{ fhservice['name'] }}!{{ fhservice.get("port", "8080") }}!{{ fhservice.get("health_endpoint", "/sys/info/health") }}
-       use generic-service
-       hostgroup_name {{ ",".join(fhservice['hostgroups']) }}
-       notes This server failed to get a successful response from the {{ fhservice['name'] }} health endpoint. Check the response code & body and ensure the service and its dependencies are running and configured correctly.
-       contact_groups rhmapadmins
-}
+{% for hostgroup in rhmap_hostgroups.split(',') %}
```

Comment:
@mikenairn I assure you that we all agree this seems stupid, but the reason here is rather important.  If a nagios is added to a project with both an mbaas and a core, both of which have some components like fh-messaging, the mbaas fh-messaging needs to be verified separately of the core fh-messaging (because they have unique host names).  In order for nagios to display both with different check commands, it needs to have unique descriptions.  That's why we are putting the mbaas or core in the service description.